inc/namespace.php

```diff
@@ -151,6 +151,14 @@ function get_aws_sdk() : Sdk {
 			'secret' => AWS_SECRET,
 		];
 	}
+
+	/**
+	 * Filter params before new instance of Sdk is created.
+	 *
```

Comment:
Found a link documenting the options: 
```suggestion
	 *
	 * See the AWS SDK docs for all available options:
	 * https://docs.aws.amazon.com/aws-sdk-php/v3/api/class-Aws.AwsClient.html#___construct
	 *
```